src/renderer/components/input/input.tsx

```diff
@@ -173,22 +175,18 @@ export class Input extends React.Component<InputProps, State> {
             error => this.getValidatorError(value, validator) || error,
           ),
         );
-      } else {
-        if (!validator.validate(value, this.props)) {
-          errors.push(this.getValidatorError(value, validator));
-        }
```

Comment:
This was causing to doubling same error message:
![image](https://user-images.githubusercontent.com/6377066/171930477-5ce73335-b448-47a8-a699-ad9c165c1f34.png)

Reply:
I thought I also fixed this... Ah it is part of https://github.com/lensapp/lens/pull/5403 which hasn't been merged yet.